documentation/API-design-guidelines.md

```diff
@@ -1715,7 +1746,11 @@ Remark: This action will trigger a subscription-ends event with terminationReaso
 | initialEvent           | boolean            | Set to true by API consumer if consumer wants to get an event as soon as the subscription is created and current situation reflects event request. Example: Consumer request Roaming event. If consumer sets initialEvent to true and device is in roaming situation, an event is triggered. Up to API project decision to keep it. | optional    |
 
 
-Subscription status value table:
+**Note** on combined usage of initialEvent and subscriptionMaxEvents: 
+Unless explicitly decided otherwise by the API WG,  if an event is triggered following initialEvent set to `true`, this event will be counted towards subscriptionMaxEvents (if provided).
```

Comment:
Aligning with [Governance definitions](https://github.com/camaraproject/Governance/blob/main/ProjectStructureAndRoles.md#project-structure)

```suggestion
Unless explicitly decided otherwise by the API Sub Project,  if an event is triggered following initialEvent set to `true`, this event will be counted towards subscriptionMaxEvents (if provided).
```